workloads/kube-burner-ocp-wrapper/README.md

```diff
@@ -46,6 +46,11 @@ Or increase the benchmark timeout (by default 3h):
 $ EXTRA_FLAGS="--timeout=5h" ITERATIONS=500 WORKLOAD=cluster-density-v2 ./run.sh
 ```
 
+Or change the deletion strategy during churn, It is recommended for running churn cycles on large(500) scale clusters
```

Comment:
I would choose to talk about deletion strat OR cycles. Adding both to the same line makes it a touch confusing

Reply:
updated it